mocks/mocks.go

```diff
@@ -248,6 +248,10 @@ func (sa *StorageAuthorityReadOnly) GetCertificateStatus(_ context.Context, req
 	}
 }
 
+func (sa *StorageAuthorityReadOnly) SetCertificateStatusReady(ctx context.Context, req *sapb.Serial, _ ...grpc.CallOption) (*emptypb.Empty, error) {
+	return nil, berrors.InternalServerError("not implemented")
```

Comment:
berrors.InternalServer was deprecated in #6050. I believe we're supposed to return a normal error now.